src/main/java/com/avail/utility/evaluation/Combinator.java

```diff
@@ -35,31 +35,14 @@
 import kotlin.Unit;
 import kotlin.jvm.functions.Function0;
 import kotlin.jvm.functions.Function1;
+import kotlin.jvm.functions.Function2;
 
 /**
- * Utility for applying a Continuation1 recursively.
+ * Utility for applying a functions recursively.
```

Comment:
function

Reply:
Actually, it's a "Utility for allowing a function body to recursively invoke, or otherwise reference the function itself."

Reply:
Updated the whole message to say:

```
Utilities for applying functions recursively.
```